README.md

```diff
@@ -417,6 +417,14 @@ channel.bind('my-event', function () {
 }, { name: 'Pusher' });
 ```
 
+For client-events on presence channels, bound callbacks will be called with an additional argument, an object containing the `user_id` of the user who triggered the event
+
+```
+presenceChannel.bind('client-message', function (data, metadata) {
+  console.log('received message', data.message, 'from', metadata.user_id);
```

Comment:
What does `data` look like? I couldn't figure out whether it would always have a `message` field...

Reply:
Data comes from a triggered event which name, in this case, is `client-message` and could be the type of object with a defined key-value pair.

Reply:
It wouldn't always have a message field. It could be an `Object` an `Array` or a `String`. I should update this example as it could be mislreading